tests/functionnal/Dropdown.php

```diff
@@ -1248,6 +1248,188 @@ public function testGetDropdownUsers($params, $expected)
         $this->array($result)->isIdenticalTo($expected);
     }
 
+    /**
+     * Summary of testShow
+     */
+    public function testShow()
```

Comment:
This kind of test is not maintainable. Not sure it has sense to add it.

@trasher @orthagh your opinion ?

Reply:
Agree.

If a test should be done on html output, i guess i'll go with a dom crawler (like `symfony/dom-crawler` we already have) to check presence of some tags (`select[multiple]` for example).
But doing an equality of all generated html seems a bad idea

Reply:
I agree with @orthagh; the test cans be kept, but not on an entire html output